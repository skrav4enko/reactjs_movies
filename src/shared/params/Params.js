import React from 'react';
import './Params.scss';

function Params(props) {
  const { value, dimension } = props;

  return (
    <div className="params">
      <span className="params__value">{value}</span>
      <span>{dimension}</span>
    </div>
  );
}

export default Params;
