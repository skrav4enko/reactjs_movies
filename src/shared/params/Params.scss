.params {
  display: inline-block;

  &__value {
    color: #f56261;
    font-size: 24px;
    margin-right: 10px;
  }
}
