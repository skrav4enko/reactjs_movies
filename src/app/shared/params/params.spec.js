/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import Params from '.';

describe('Params', () => {
  let testProps;

  beforeAll(() => {
    testProps ={
      dimension: 'test',
      value: 5,
    }
  });

  it('should match snapshot', () => {
    const component = shallow(<Params {...testProps} />);
    expect(component).toMatchSnapshot();
  });
});

