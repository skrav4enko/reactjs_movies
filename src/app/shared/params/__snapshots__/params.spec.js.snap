// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Params should match snapshot 1`] = `
<div
  className="params"
>
  <span
    className="params__value"
  >
    5
  </span>
  <span>
    test
  </span>
</div>
`;
